src/components/webviews/jsInteractions/jsCozyInjection.ts

```diff
@@ -21,13 +21,16 @@ const makeMetadata = routeName => {
   })
 }
 
-export const jsCozyGlobal = (routeName, isSecureProtocol) => `
+export const jsCozyGlobal = (
+  routeName?: string,
+  isSecureProtocol?: boolean
+): string => `
   if (!window.cozy) window.cozy = {}
   window.cozy.isFlagshipApp = true
   window.cozy.ClientKonnectorLauncher = 'react-native'
   window.cozy.ClientConnectorLauncher = 'react-native' // deprecated
-  window.cozy.flagship = ${makeMetadata(routeName)}
-  window.cozy.isSecureProtocol = ${isSecureProtocol || 'false'}
+  window.cozy.flagship = ${routeName ? makeMetadata(routeName) : ''}
```

Comment:
Maybe whe should instead
```
  window.cozy.flagship = ${makeMetadata(routeName ?? '')}
```

Seems like having an empty `flagship` field would produce more side effect than having a wrong `immersive` data

Reply:
Updated